src/printer/OITimeToolOutputBuilder.cc

```diff
@@ -94,7 +87,16 @@ int OITimeToolOutputBuilder::encodeStatusCode() const {
             return CODE_RE_BASE + exitStatus_;
         }
     }
-    return CODES[killReason_];
+
+    switch (killReason_){
+        case KillReason::NONE: return 0;
```

Comment:
Why can't we put the code from if above here?

Reply:
Duplicated comment, I can't get used to github's review ui.

Reply:
No, it would be ugly.

Reply:
The switch is a _pure_ function from `KillReason` to `int`.